en-US/emails/2017/fundraising_thank_you.lang

```diff
@@ -0,0 +1,58 @@
+## NOTE: Thank you email for the 2016 fundraising campaign. Email screenshot: https://cloud.githubusercontent.com/assets/12228435/21836541/cfde85d2-d77a-11e6-8566-4d28f0667606.png
+
+
+# Email subject
+;Thanks!
+Thanks!
+
+
+# Preheader (only displayed in thread view https://www.campaignmonitor.com/assets/uploads/2015/08/preheader-text-on-mobile.png ), try to keep the translation around 40-50 characters, you can rephrase as needed.
```

Comment:
old habit just to make sure the parenthesis is not considered part of the link, will remove it